tests/integration/configs/configs.spec.ts

```diff
@@ -156,6 +158,199 @@ describe('config', function () {
     });
   });
 
+  describe('updateOldConfigs', function () {
+    it('should complete successfully when no old configs exist', async function () {
+      const configManager = dependencyContainer.resolve(ConfigManager);
+
+      const action = configManager.updateOldConfigs();
+
+      await expect(action).resolves.not.toThrow();
+    });
+
+    it('should update old configs with v1 schema version to v2', async function () {
```

Comment:
Shouldn't we also check that the oldConfigs were decreased (to zero) after updating?